bin/ch/WScriptJsrt.cpp

```diff
@@ -325,11 +334,46 @@ JsErrorCode WScriptJsrt::InitializeModuleInfo(JsValueRef specifier, JsModuleReco
     return JsNoError;
 }
 
-JsErrorCode WScriptJsrt::LoadModuleFromString(LPCSTR fileName, LPCSTR fileContent)
+bool WScriptJsrt::GetFullPath(LPCSTR fileName, __out char* const fullPathNarrow)
+{
+    size_t len = 0;
+    if (fileName)
+    {
+        if (_fullpath(fullPathNarrow, fileName, _MAX_PATH) == nullptr)
+        {
+            return false;
```

Comment:
It's probably cleaner to return error in FetchImportedModuleHelper when it sees false here.

---
In reply to: [125370885](https://github.com/Microsoft/ChakraCore/pull/3284#discussion_r125370885) [](ancestors = 125370885,125370693)